lib/minidoc/timestamps.rb

```diff
@@ -15,6 +15,28 @@ def timestamps!
     end
   end
 
+  def set(attributes)
+    if self.class.record_timestamps
+      super(attributes.merge(updated_at: Time.now.utc))
+    else
+      super(attributes)
+    end
+  end
+
+  def atomic_set(query, attributes)
+    if self.class.record_timestamps
+      super(query, attributes.merge(updated_at: Time.now.utc))
+    else
+      super(query, attributes)
+    end
+  end
+
+  def unset(*keys)
+    super
+
+    set(updated_at: Time.now.utc) if self.class.record_timestamps
```

Comment:
Yea, that all makes sense. I think if I had _all of_ my druthers I'd go with the touch version and keep it private for now (and not implement it for non-timestamped models) if we don't want to commit to making it a public method just to avoid this slight redundancy. But any of the options are fine with me